src/main/java/com/spotinst/sdkjava/model/bl/ocean/gke/LaunchSpecLabelsSpecification.java

```diff
@@ -0,0 +1,75 @@
+package com.spotinst.sdkjava.model.bl.ocean.gke;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import java.util.HashSet;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class LaunchSpecLabelsSpecification {
+    @JsonIgnore
+    private Set<String> isSet;
+    private String      key;
+    private String      value;
+
+    public LaunchSpecLabelsSpecification(){ isSet = new HashSet<>();
+    }
```

Comment:
include gettor and settor for isSet

Reply:
Why the getter setter is required for is set ?